code/modules/clothing/glasses/engine_goggles.dm

```diff
@@ -101,17 +101,17 @@
 
 	for(var/i in rad_places)
 		var/turf/place = i
-		if(get_dist(user, place) >= range*2)	//Rads are easier to see than wires under the floor
+		if(get_dist(user, place) >= range*5)	//Rads are easier to see than wires under the floor
 			continue
 		var/strength = round(rad_places[i] / 1000, 0.1)
-		var/image/pic = new(loc = place)
+		var/image/pic = image(loc = place)
 		var/mutable_appearance/MA = new()
-		MA.alpha = 180
 		MA.maptext = "[strength]k"
```

Comment:
While you're at it, can you add the maptext class so that it doesn't look super blurry?

```suggestion
		MA.maptext = "<span class='maptext'>[strength]k</span>"
```